static/js/profile.js

```diff
@@ -0,0 +1,645 @@
+/* Code by Milos Kozak
+
+TODO:
+	- treatment drawer must be updated to enter glycemic index
+	- update calculation to use new style of values
+	- how to handle units? store native or convert to mg/dL?
+
+*/
+	var c_profile = null;
+	var apisecret = "", storeapisecret = false;
```

Comment:
we shouldn't store the unhashed `apisecret` in memory or in localstorage, instead we should hash it right away